.gitignore

```diff
@@ -7,7 +7,6 @@ bower_components
 trash
 build
 target
-.jruby
```

Comment:
I would leave this in the .gitignore because there really is no harm in having it there. If developers still have their .jruby directory in their working directory then it will show up in `git status`. I just checked my directory and it's still there :)